apps/validate/validate.py

```diff
@@ -270,13 +272,16 @@ def _process_media(self, doc, validation_schema):
         :param doc: Article to be validated
         :param schema: Schema to validate the article
         """
+
         for field_schema in validation_schema:
             field_associations = field_schema if field_schema != 'feature_media' else 'featuremedia'
-            media_field = self._get_media_field(field_schema, field_associations, doc)
-            if media_field and isinstance(doc['associations'][media_field], MutableMapping):
-                doc[field_schema] = doc['associations'][media_field]
-                if not doc.get('feature_media', None) is None and 'description_text' in doc['feature_media']:
-                    doc['media_description'] = doc['associations']['featuremedia']['description_text']
+            media_fields = self._get_media_fields(field_schema, field_associations, doc)
+            for media_field in media_fields:
+                if media_field and isinstance(doc['associations'][media_field], MutableMapping):
+                    doc[field_schema] = doc['associations'][media_field]
+                    if not doc.get('feature_media', None) is None and 'description_text' in doc['feature_media']:
+                        doc['media_description'] = doc['associations']['featuremedia']['description_text']
+                    del doc['associations'][media_field]
```

Comment:
why is it removing the field?

Reply:
to skip validation on it I guess..

Reply:
The field that is validated is copied on doc root at line 281 and this will be validated.
There is a validation for associations that treat all items as media and requests description_text, alt_text and other media related fields.